spec/jobs/appointment_notification/worker_spec.rb

```diff
@@ -17,6 +17,7 @@
   let(:callback_url) { "https://localhost/api/v3/twilio_sms_delivery" }
 
   before do
+    Flipper.enable(:appointment_reminders)
```

Comment:
Do you need to disable it after the test?

Reply:
We use the DB for flipper store, so I think the flipper changes get rolled back after every spec.

Reply:
The db tx rollback would take care of this, I think. I'm merging this now so we don't send out reminders today.

Reply:
They are reset for every spec, as flipper is part of database state.